test/config.json

```diff
@@ -900,6 +900,15 @@
 			"RetryMax": 1,
 			"Tags": []
 		},
+		"vtgate_grpc_server_auth_static": {
+			"File": "unused.go",
+			"Args": ["vitess.io/vitess/go/test/endtoend/vtgate/grpc_server_auth_static"],
+			"Command": [],
+			"Manual": false,
+			"Shard": "vtgate_general_heavy",
```

Comment:
I'm not sure if this is the right shard to use. Any guidance would be appreciated.

Reply:
This shard looks as good as any. I checked the run of this shard as well. It takes about 13 minutes with this new test, so there is no risk of flakiness because of timeouts. It should be okay.